src/index.js

```diff
@@ -51,6 +51,8 @@ class Uploader {
     }
 
     static prettyFileSize(size) {
+        if(!size) { return '0KB'; }
```

Comment:
This seems more like an error to me. Maybe we should throw here?

Reply:
lets talk about it when you come back